### DISCUSSION TITLE
usman/how-to-use-mongoose-with-next-js-for-mongodb

### DISCUSSION THREAD

**Discussion by giscus** [genicsblog/comments#129] [Announcements]
# usman/how-to-use-mongoose-with-next-js-for-mongodb

Learn using Mongoose for MongoDB by building a hands-on project!

https://genicsblog.com/usman/how-to-use-mongoose-with-next-js-for-mongodb

**Comment by Arshroop-Saini** [comment]
Learned a lot!
